### PR TITLE
Adopt governance and workflow structure from Grafana

### PR DESCRIPTION
**Pull request by RichiH**
SECURITY.md is WIP and will be added once it's released for Grafana.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
-* **Supermajority Vote**: More than two thirds vote in favor
+* **Projects**: A single repository in the Grafana GitHub organization and listed below is referred to as a project:
+  * metrictank
+
+* **The Metrictank project**: The sum of all activities performed under this governance, concerning one or more repositories or the community.
 
 ## Values
 
-The Metrictank developers and community are expected to follow the values defined in the [Metrictank Code of Conduct][coc]. Furthermore, the Metrictank community strives for kindness, giving feedback effectively, and building a welcoming environment. The Metrictank Team generally decide by [lazy consensus](#consensus) and only resort to conflict resolution by a [majority](#majority-vote) vote if consensus cannot be reached.
+The Metrictank developers and community are expected to follow the values defined in the [Code of Conduct][coc]. Furthermore, the Metrictank community strives for kindness, giving feedback effectively, and building a welcoming environment. The Metrictank developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
```
Comment (Dieterbe) [inline GOVERNANCE.md:16]: consensus means we need a list of who the developers are. is developers the same as maintainers?

Comment (RichiH) [reply GOVERNANCE.md:16]: Yes; this is clear in the consensus & voting sections which define how to actually do things. Valid point, but I would prefer to not change it, or change it across all projects after a vote has passed either way.

**Comment by robert-milan** [comment]
Before merging what else needs to be done here?

1. We need a MAINTAINERS.md file
2. We need a CODE_OF_CONDUCT.md file

Anything else?

**Comment by RichiH** [comment]
Two examples for new MAINTAINERS.

Grafana
````
@torkelo is the main/default maintainer, some parts of the codebase have other maintainers:

* Backend:
  * @bergquist
* Plugins:
  * @ryantxu
* UX/UI:
  * @davkal
````

Loki
````
@slim-bean is the main/default maintainer.
````

**Comment by Dieterbe** [comment]
as a starting point, something like the below? @fkaleo have we carved out a section for you yet?

```
@Dieterbe is the main/default maintainer, some parts of the codebase have other maintainers:

* idx:
  * @replay , @robert-milan, @woodsaj, @shanson7   
* expr:
  * @Dieterbe, @robert-milan, @shanson7  
* api:
  * @Dieterbe, @replay, @robert-milan, @shanson7  
```

**Pull request merged**